### ISSUE TITLE
Dragging and dropping files in the Editor is failing

### ISSUE THREAD

**Issue by Martijndr** [spyder-ide/spyder#3670]
## Description of problem

I've just upgraded from Spyder 2.x to Spyder 3.x, and noticed it is no longer possible to drag-and-drop files to open them. Not sure if this is intentional functionality or not, but opening files on windows now has become a bit clunky.

**Try the following steps to reproduce the problem:**

1. Have Spyder 3.0.x version installed
2. Open Spyder & drag drop .py file into editor

**Expected and given output:**

Based on the previous version of Spyder, I expect dragging and dropping in a file to open that file in the editor, instead it's copying the file location.

## Versions and main components

* Spyder Version: 3.0.1
* Python Version: 3.5
* Operating system: Windows 10

**Comment by ccordoba12** [comment]
@malliwi88, please work on this one.

**Comment by malliwi88** [comment]
Hi @ccordoba12, 

It doesn't look like a bug, I try it on Windows 64 bits with the same spyder version (3.0.1) and all is working  fine.

![win10 corriendo - oracle vm virtualbox_014](https://cloud.githubusercontent.com/assets/5307894/20196304/1cb29fda-a768-11e6-9d19-396162196d20.png)

Could @Martijndr, create another conda environment and test it once time more?

**Comment by ccordoba12** [comment]
@dalthviz, could you also test this one in your computer, please?

@Martijndr, how did you install Spyder?

**Comment by dalthviz** [comment]
Hi everyone. I tested the drag-and-drop function and everything seems fine (the file with .py extension opens in the editor). My specs are Windows 8 64 bits, Spyder 3.0.1 and python 3.5

**Comment by Martijndr** [comment]
Hi everyone. Thanks for picking this up - I can confirm it's happening to me and all my colleagues also using spyder.
I installed spyder (and python) using Anaconda.
My specs are: Windows 10 64-bits, Spyder 3.0.1 and python 3.5

**Comment by ccordoba12** [comment]
I'll test on my virtual machine to do an extra verification.

@Martijndr, we can't fix this problem unless we can reproduce it :-)

**Comment by ccordoba12** [comment]
I can't reproduce this issue either, sorry.

**Comment by ccordoba12** [comment]
I think the problem is your files are not encoded in Unicode and have non-ascii characters on them. Only in that case drag-and-drop fails (although I couldn't reproduce that file names are copied to the Editor).

This is going to be fixed in Spyder 3.1 with PR #3640.

**Comment by songxiaocheng** [comment]
Hi, I think I've encountered the same problem: when I drag my `.py` files into spyder, the pointer becomes a "No" symbol. My system is win10 x64, with Anaconda x64 and spyder 3.2.8.
Finally I solved it by right click -> `property` -> `compatibility `-> uncheck the option `Run this program as an administrator`. This is checked by myself before instead of default.  So, don't run spyder as an administrator, which may solve the drag-and-drop fails problem.
